pipelines/streaming/README.md

```diff
@@ -0,0 +1,130 @@
+The
+[streaming](https://github.com/google/fhir-data-pipes/tree/master/pipelines/streaming)
+directory contains code to continuously listen for changes to an underlying
+OpenMRS MySQL database using
+[Debezium](https://debezium.io/documentation/reference/1.2/connectors/mysql.html).
+
+The Debezium-based streaming mode provides real-time downstream consumption of
+incremental updates, even for operations that were performed outside the OpenMRS
+API, e.g., data cleaning, module operations, and data syncing/migration. It
+captures incremental updates from the MySQL database binlog then streams both
+FHIR and non-FHIR data for downstream consumption. It is not based on Hibernate
+Interceptor or Event Module; therefore, all events are captured from day 0 and
+can be used independently without the need for a batch pipeline. It also
+tolerates failures like application restarts or crashes, as the pipeline will
+resume from the last processed offset.
+
+Streaming mode works only with OpenMRS; support for other data sources is not
+planned.
```

Comment:
Please consider adding a note that this pipeline is in maintenance mode and is not actively being developed (as we have added the incremental [micro-]batch pipeline). There is also a problem with duplicated resources in Parquet files which should be mentioned somewhere as it is one of the main reasons we have not focused on this mode.

Reply:
Note was added to the top of the file.